src/main/java/com/microsoft/lst_bench/telemetry/JDBCTelemetryRegistry.java

```diff
@@ -109,7 +109,8 @@ public void flush() {
       eventsStream = Collections.synchronizedList(new ArrayList<>());
       LOGGER.info("Events flushed to database.");
     } catch (SQLException e) {
-      LOGGER.error("Error while flushing events to database", e);
+      LOGGER.error("Error while flushing events to database");
```

Comment:
Nit. Maybe keep passing the exception to the logger so the stacktrace is printed; having complete information in the log makes debugging easier.

Reply:
Hi @jcamachor. Thanks for the review.
I am following this principle here: "Don’t just Log & Rethrow" [ref](https://belief-driven-design.com/all-you-ever-wanted-to-know-about-java-exceptions-63d838fedb3/)

The reason for not logging the error stack here is log duplication (_The upstream code has no way to know if an exception was already logged. And it might be propagated through multiple frames_).
This method is not taking action actions other than  passing the exception through to its caller. Doing so allows logging the error stack trace at one place where the exception is getting handled and based on the severity that the caller assigns. WDYT?

Reply:
I'm fine either way, that's why I had +1.
But I'd argue that then probably it's better to remove the error log message all together: Having an error message in the logs without enough details to understand what failed is confusing imo.